### PR TITLE
Clarify the text about ghosts and puppets in Matrix

### PR DESCRIPTION
**Pull request by DataBeaver**
Muistin / sain viimein aikaseks tehä tän selvennyksen mistä oli puhetta jo pari kuukautta sitte.

### CODE REVIEW

**Review by tulir — APPROVED** [review]

**Review by Mikaela — APPROVED** [review]
Thanks for the PR, I will likely merge tomorrow morning if not later in the evening.

The wording turns a bit more technical and I am having some headache/brainfog and as I pinged a couple of people in private for second opinion, I am going to give them a chance to say something before merging this.

**Comment by DataBeaver** [comment]
Let me know if you want me to have another go at rewriting it in a less technical way.  And hopefully not take two months to do it this time...

**Comment by Mikaela** [comment]
Thanks again and also thanks to everyone who reviewed either here or on Matrix 💜

**Pull request merged**
